src/nxdoc/nuxeo-server/administration/database-configuration/documentdb.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Amazon DocumentDB
+labels:
+    - database
+    - documentdb
+tree_item_index: 700
+
+---
+The Nuxeo Platform supports Amazon DocumentDB
+
+## Configuration
+
+Since Nuxeo 2023.15, using the `mongodb` template, you can use `DocumentDB` as a `DBS` solution with some limitations:
+
+- fulltext is not supported as `DocumentDB` has various limitations in that aspect (english only, no term exclusion, semantics differences...) see [Differences With MongoDB](https://docs.aws.amazon.com/documentdb/latest/developerguide/text-search.html#text-index-mongo-diff) for more details.
+You need to disable it like with the following nuxeo configuration:
+```properties
+nuxeo.vcs.fulltext.search.disabled=true
+```
+- depending on your cluster size, you might encounter concurrent cursors limitations. It will result with a message similar to this:
+```console
+Cannot open a new cursor since too many cursors are already opened
+```
```

Comment:
This is a configuration section, I expect a list of options to configure not a list of possible problem,
this should be a in a Limitation section.

Reply:
I followed the layout you proposed in the review comment. Makes more sense indeed!